content/doc/administration/requirements/java.adoc

```diff
@@ -7,27 +7,32 @@ There are separate run and job execution requirements for Jenkins installations.
 
 ## Running Jenkins
 
-Java 8 or Java 11 are required for running modern versions of Jenkins. Upgrading an existing Jenkins setup from Java 8 to Java 11? See the link:/doc/administration/requirements/upgrade-java-guidelines[upgrade guidelines].
+Jenkins requires Java 11 or 17 since Jenkins 2.357 and LTS 2.361.1.
+link:/blog/2022/06/28/require-java-11/[Read more about this in the announcement blog post.]
 
-Java 11 Docker installation instructions are included in link:/doc/book/installing/docker/#downloading-and-running-jenkins-in-docker["Downloading and running Jenkins in Docker"]
+[NOTE]
+.upgrading Java version to 11
```

Comment:
Why? It looks like this:

<img width="1386" alt="Screenshot 2022-09-19 at 08 57 09" src="https://user-images.githubusercontent.com/1831569/190964314-d16de2f9-7b9c-4a7c-bf44-7fde44473c2a.png">

Reply:
> Why? It looks like this:
> 
> <img alt="Screenshot 2022-09-19 at 08 57 09" width="1386" src="https://user-images.githubusercontent.com/1831569/190964314-d16de2f9-7b9c-4a7c-bf44-7fde44473c2a.png">

I liked the italic text on the admonition.  I thought it was the right level of additional emphasis without the high visual impact that comes from a heading.  I don't know if that's exactly why @kmartens27 did it that way, but I thought it looked reasonable in the context of the admonition.

Reply:
I had updated this to an admonition block as you had suggested previously @daniel-beck , would you be able to clarify what you mean by why?
![Screen Shot 2022-09-19 at 9 13 53 AM](https://user-images.githubusercontent.com/99040580/191025518-4ca9cad2-9b7d-4e23-9476-ee1c43544864.png)

Reply:
I cannot speak to the reason why Daniel did not like this (nor did he explicitly state it), but I do not think the lowercase "u" looks right. Regardless of whether one is using title case or sentence case, the first letter of a line should always be capitalized.

Reply:
> would you be able to clarify what you mean by why?

The specific line I am commenting on is unnecessary.